FinniversKit/Sources/Components/NMPInfobox/NMPInfoboxView+InformationType.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
```

Comment:
Top template comments should be deleted.

Reply:
Sorry I don't follow...In all files?

Reply:
Yes, we have no need for these autogenerated comments. Or wait, this is public Github, might be different here? @osanoj

Reply:
Yeah all the other files has the comments so that's why I kept them. I mean, I'll happily delete them, as long as we are consistent 😄

Reply:
Yes, we usually keep them on public GitHub repos. 

And remove them on internal repos, just to make sure to keep everyone one their toes 😜